src/custom-element.js

```diff
@@ -119,6 +119,8 @@ export function upgradeOrRegisterElement(win, name, toClass) {
     const element = stub.element;
     if (element.tagName.toLowerCase() == name) {
       tryUpgradeElementNoInline(element, toClass);
+      // Remove element from array.
+      stubbedElements.splice(i--, 1);
```

Comment:
What if `tryUpgradElementNoInline` fails, do also want to remove it from `stubbedElements`? 
Another thing is removing element and decrease the i here is not super straightforward, do you think that starting from the end of the array would be better?